jsapp/js/dataInterface.ts

```diff
@@ -632,7 +636,14 @@ export interface AssetResponse extends AssetRequestObject {
   }
   deployment__active: boolean
   deployment__data_download_links?: {
-    [key in ExportTypeName]: string | undefined
+    csv_legacy: string
```

Comment:
Same as above

Reply:
After I've added the mock file (content of the file is real server responses), it turned out that not all of `ExportTypeName`s are always present, so I changed the interface to be more precise in that